src/Compilers/Core/Portable/Operations/IIncrementExpression.cs

```diff
@@ -19,9 +17,29 @@ public interface IIncrementExpression : IOperation, IHasOperatorMethodExpression
         IOperation Target { get; }
 
         /// <summary>
-        /// Kind of increment.
+        /// <code>true</code> if this is a decrement expression.
+        /// <code>false</code> if this is an increment expression.
+        /// </summary>
+        bool IsDecrement { get; }
```

Comment:
>bool IsDecrement { get; } [](start = 8, length = 25)

I think we should consider to simply use distinct kind for decrement rather than having a special property. IIncrementExpression could have two different kinds. Something to discuss at the next design meeting.

Reply:
It was originally a separate kind, but we decided to remove it when you were OOF. Lets discuss it one more time next meeting - I will file a tracking issue.

Reply:
Filed https://github.com/dotnet/roslyn/issues/21748